src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -1002,6 +1019,8 @@ fun PrisonApiPrisoner.toPrisonerSearchPrisoner() = PrisonerSearchPrisoner(
     ?: this.sentenceDetail.topupSupervisionExpiryDate,
   paroleEligibilityDate = this.sentenceDetail.paroleEligibilityOverrideDate
     ?: this.sentenceDetail.paroleEligibilityDate,
+  postRecallReleaseDate = this.sentenceDetail.postRecallReleaseOverrideDate
+    ?: this.sentenceDetail.postRecallReleaseDate,
```

Comment:
we weren't setting this so it was always null. 
This meant on any time a sentence change occurred we'd null out the PRRD and LSD